.github/workflows/zxc-e2e-test.yaml

```diff
@@ -163,7 +163,7 @@ jobs:
         if: ${{ runner.os == 'linux' && (inputs.npm-test-script == 'test-e2e-node-local-build' || inputs.npm-test-script == 'test-e2e-node-add') && !cancelled() && !failure() }}
         run: |
           cd ..
-          git clone https://github.com/hashgraph/hedera-services.git  --branch v0.53.2
+          git clone https://github.com/hashgraph/hedera-services.git  --branch release/0.54
```

Comment:
I would rather it be a tag rather than a branch to avoid a random build failure.